backend/apps/security/views.py

```
@@ -120,6 +122,12 @@ class AutoFixPRViewSet(viewsets.ModelViewSet):
     permission_classes = [permissions.AllowAny]
 
 
+class ProjectDependencyViewSet(viewsets.ModelViewSet):
+    queryset = ProjectDependency.objects.all()
+    serializer_class = ProjectDependencySerializer
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)